codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -350,7 +349,7 @@ private static void writeChildNode(FlowState state) {
         FlowState(Http2Stream stream, int initialWindowSize) {
```

Comment:
@Scottmitch hmm well it's not used in tests and IMHO we should not care about synthetic methods at all, unless I am missing something. WDYT?